docs/bots-guide.md

```diff
@@ -253,8 +252,23 @@ running it manually.
    The standard output of the bot server will be logged to the path in
    your *supervisord* configuration.
 
+### Running a bot in your Zulip Development Environment
```

Comment:
Since this section is about running a bot, we probably want it to be a subsection of "How to run a bot".